@here/harp-omv-datasource/lib/OmvUtils.ts

```diff
@@ -7,8 +7,8 @@
 /**
  * @hidden
  */
-export function isArrayBufferLike(data: any): data is ArrayBufferLike {
-    return data instanceof ArrayBuffer || data instanceof SharedArrayBuffer;
+export function isArrayBuffer(data: any): data is ArrayBufferLike {
```

Comment:
should return `data` is ArrayBuffer`

Reply:
`ArrayBufferLike` already only contains `ArrayBuffer`, I don't know why it was implemented this way, I thought it didn't impact the problem so I've let it this way. It could be deleted entirely yes. Should I do it?

Reply:
Yes, this is quite confusing. Please remove it.

Reply:
It is actually in the es5 type definition, could not remove. Implemented what Oleksii suggested instead.